src/select/select.js

```diff
@@ -130,6 +135,14 @@ class Select extends React.Component<PropsT, SelectStateT> {
     this.clearValue(event);
   };
 
+  handleClickOutside = (event: Event) => {
+    const isFocused = this.state.isFocused || this.state.isPseudoFocused;
+    // $FlowFixMe
```

Comment:
this may help? https://stackoverflow.com/questions/48433096/react-flowtype-node-contains-event-target

Reply:
Hm, it seems we can't cast to `any` because there's this eslint rule `flowtype/no-weak-types`?

Reply:
cool, no worries